kits/multisig/contracts/MultisigKit.sol

```diff
@@ -58,8 +75,8 @@ contract MultisigKit is BetaKitBase {
         uint256 multisigSupport = neededSignatures * 10 ** 18 / signers.length - 1;
         voting.initialize(
             token,
-            multisigSupport,
-            multisigSupport,
+            uint64(multisigSupport),
+            uint64(multisigSupport),
```

Comment:
Could we use the [`Uint256Helpers`](https://github.com/aragon/aragonOS/blob/dev/contracts/common/Uint256Helpers.sol) here and in the democracy kit?